File: app/src/main/java/com/example/abcmobile/activity/M_Info.java
package com.example.abcmobile.activity;

import android.content.ClipData;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import com.example.abcmobile.R;

public class M_Info extends AppCompatActivity {

    private TextView tv_info_saldo;
    private TextView tv_favorit;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.m_info);

        tv_info_saldo = findViewById(R.id.tv_info_saldo);
        tv_favorit = findViewById(R.id.tv_favorit);

        tv_info_saldo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(M_Info.this, InfoAccount.class));
            }
        });

        tv_favorit.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(M_Info.this, Favorit.class));
            }
        });
    }

    @Override
    public void onBackPressed() {
        finish();
    }

}

File: app/src/main/java/com/example/abcmobile/activity/MInfo.kt
package com.example.abcmobile.activity

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import com.example.abcmobile.R

class MInfo : AppCompatActivity(), View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.m_info)

        val btnInfoSaldo: Button = findViewById(R.id.btn_info_saldo)
        btnInfoSaldo.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when(v.id) {
            R.id.btn_info_saldo ->{
                val moveIntent = Intent(this@MInfo,InfoSaldo:: class.java)
                startActivity(moveIntent)
            }
        }
    }
}